Minor updates on some section with outdated information (#24752)
DataDog/documentation: content/en/tracing/trace_explorer/query_syntax.md

{{< img src="tracing/app_analytics/search/create_facet.png" style="width:50%;" alt="Create Facet" style="width:50%;">}}

> Once you create a new facet, it will be available in the facet panel for quick filtering and basic analytics.

Reviewer: ```suggestion
After you create a new facet, it is available in the facet panel for filtering and basic analytics.
```
